config/locales/de.yml

```diff
@@ -1,7 +1,7 @@
 # German strings go here for Rails i18n
 de:
-  error_unable_to_update_project_gtt_settings: Die GTT-Einstellungen des Projekts können
-    nicht aktualisiert werden (%{value})
+  error_unable_to_update_project_gtt_settings: Die GTT-Einstellungen des Projekts
+    können nicht aktualisiert werden (%{value})
```

Comment:
Also new lines (without back slash) at here.
Maybe this comes from Weblate setting ?